menyer/Vegancorner/style.css

```diff
@@ -0,0 +1,109 @@
+@import url('https://fonts.googleapis.com/css2?family=Oswald&display=swap');
+
+*   {
+    margin: 0;
+    padding: 0;
+    font-family: 'Oswald', sans-serif;
+}
+
+body    {
+    background-color: rgb(0, 0, 0);
+}
+
+/*TOP CONTENT*/
+
+header  {
+    width: 100%;
+    height: 140px;
+    border: 1px solid black;
+
+    background-image: url(images/header.jpg);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+header h1   {
+    text-align: center;
+    font-size: 50px;
+    background-color: rgba(255, 247, 247,0.5);
+    text-shadow: 0 0 10px white;
+}
+
+header h3   {
+    text-align: center;
+    background-color: rgba(255, 247, 247,0.5);
+    text-shadow: 0 0 10px white;
+}
+
+/*NAVIGATION BAR*/
+
+nav {
+    float: left;
+    position: fixed;
+    width: 200px;
+    height: 1500px;
+    border: 1px solid black;
+    background-color: rgb(134, 40, 24);
+}
+
+nav ul  {
+    list-style-type: none;
+    line-height: 100px;
+}
+
+nav li  {
+    border: 1px solid black;
+    text-align: center;
+}
+
+nav a    {
+    text-decoration: none;
+    font-size: 30px;
+    color: black;
+    padding: 30px 20px 30px 20px;
+
+}
+
+nav li:hover    {
+    background-color: rgba(196, 45, 19, 0.5);
+}
+
+/*MAIN CONTENT*/
+
+main    {
+    position: relative;
+    left: 110px;
+    width: 750px;
+    margin-left: 155px;
```

Comment:
Hardcoded values might work well for your screen but can look very strange on others. I recommend using %.